SQRLDotNetClientUI/DB/DBContext/SQRLDBContext.cs

```diff
@@ -20,13 +53,14 @@ public class SQRLDBContext: DbContext
 
         protected override void OnConfiguring(DbContextOptionsBuilder options)
         {
-            var directory = Path.GetDirectoryName(AppContext.BaseDirectory);
-            Log.Information($"DB Directory: {directory}");
-            if (!File.Exists(Path.Combine(directory, "sqrl.db")))
-                directory = "";
-            Console.WriteLine($"{directory}: {File.Exists(Path.Combine(directory, "sqrl.db"))}");
+            if (!Directory.Exists(Path.GetDirectoryName(PathConf.FullClientDbPath)))
```

Comment:
Couldn't we simply use `PathConf.ClientDbPath` here instead of `Path.GetDirectoryName(PathConf.FullClientDbPath)`?

Reply:
yup!